frontend/tsconfig.json

```diff
@@ -18,7 +18,11 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react"
+    "jsx": "react",
+    "typeRoots": [
+      "src/types",
+      "node_modules/@types"
+    ]
```

Comment:
This enables us to provide our own custom typings when needed. Useful when consuming npm libraries not written in TypeScript.

Reply:
I was wondering how to do this